runtime/libcn/dune

```diff
@@ -16,3 +16,9 @@
   (libcn.a as runtime/libcn.a))
  (section lib)
  (package cn))
+
+(install
+ (files
+  (libexec/cn-runtime-single-file.sh as runtime/libexec/cn-runtime-single-file.sh))
```

Comment:
https://dune.readthedocs.io/en/stable/reference/dune/install.html#including-files-in-the-install-stanza